data/containers/kubectl/deployment.yml

```
@@ -16,7 +16,9 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: nginx
+        # The testing.registry must be registered in k8s
+        # as private registry and point to REGISTRY variable.
+        image: testing.registry/library/nginx
```

Why do we need to use `testing.registry/library/nginx` and not simply `nginx`?

---

Because `nginx` expands to `docker.io/library/nginx` which is then downloaded from `docker.io` which has API limits.

---

Ahhh yes right. Mind adding a comment there then?, e.g.

```suggestion
        # Use custom registry to avoid API rate limits on docker.io
        image: testing.registry/library/nginx
```

---

No new VR needed for this change IMHO.